doc/usermanual/lighttable/panels/history.xml

```diff
@@ -47,6 +47,8 @@
         Copy the history stack of the selected image. You will be prompted for which items are
         to be include. If more than one image is selected, the history stack is taken from the
```

Comment:
Copy the history stack of the selected image. You will be prompted for which items are to be included.